File: frontend/lib/models/club_appeal_request_dto.dart
class ClubAppealRequestDto {
  final int? clubId;
  final String type;
  final String message;
  final int? requestId;

  const ClubAppealRequestDto({
    this.clubId,
    required this.type,
    required this.message,
    this.requestId,
  });

  Map<String, dynamic> toJson() => {
        'clubId': clubId,
        'type': type,
        'message': message,
        'requestId': requestId,
      };
}

File: backend/CoreService/src/main/java/ru/bowling/bowlingapp/DTO/ClubAppealRequestDTO.java
package ru.bowling.bowlingapp.DTO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ClubAppealRequestDTO {
    private Long clubId;
    private NotificationEventType type;
    private String message;
    private Long requestId;
}
